sections/appendix/process.tex

```diff
@@ -0,0 +1,30 @@
+\subsection{Project Process}
+\begin{itemize}
+	\item Initial Idea - Combine LDA and PageRank (clustering)
+	\item Preprocessing \& LDA implementation
+	\item Adjacency matrix construction
+	\begin{itemize}
+		\item challenges with calculation time
+		\item focus on more restricting preprocessing and thresholds
+	\end{itemize}
+	\item Bad results
+	\item Discovery of related papers
+	\item Found out our model didn't converge
+\end{itemize}
+
+Our initial idea with the project was to combine \gls{lda} with a PageRank algorithm to see if this could improve the information retrival results of \gls{lda}.
+
+We began by implementing some initial preprocessing methods and using an \gls{lda} implementation by gensim\todo{ref}.
+
+After this we only needed to combine \gls{lda} with a PageRank implementation. But in order to do so, we first needed to construct an adjacency matrix. This is where we started to run into trouble as constructing a large adjecency matrix based a similarity measure took extremely long time, which forced us to make changes to the way we constructed the adjacency matrix as well as changes to the earlier phases: preprocessing and \gls{lda}.
+We made the preprocessing more aggressive and implemeneted more preprocessing methods, to contenually make the dataset smaller while keeping the relevant data.
+We also implemented thresholds to the $\theta$ and $beta$ matrixes made by the \gls{lda} model. All values under the normal destribution values were set to 0, so that we could work more efficiently with sparse matrixes.
```

Comment:
```suggestion
We also implemented thresholds to the $\theta$ and $beta$ matrixes made by the \gls{lda} model. 
All values under the normal distribution values were set to 0 so that we could work more efficiently with sparse matrixes.
```